pkg/services/object/get/service.go

```diff
@@ -88,10 +117,18 @@ func WithLogger(l *zap.Logger) Option {
 	}
 }
 
+// WithoutAssembly returns option to disable object assembling.
+func WithoutAssembly() Option {
```

Comment:
not used? do we need `c.assembly` at all?

Reply:
nope, this is a rebase mistake

Reply:
to be fixed in this PR?